### DISCUSSION TITLE
v11 | Notifications `action` prop and rich text are being deprecated

### DISCUSSION THREAD

**Discussion by dakahn** [carbon-design-system/carbon#8533] [Ideas]
For screen reader accessibility notifications cannot contain interactive content or rich text due to limitations with how `aria-live` and `role="alert"` are interpreted by screen readers

- content of `role="alert"` is read as a string, so rich text (headings, emphasis etc) are lost and not accessible
- interactive elements (buttons, links etc) are inaccessible via keyboard and screen reader since notifications do not steal focus

**Comment by tay1orjones** [comment]
Related https://github.com/carbon-design-system/carbon/pull/8560
